docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -143,12 +143,56 @@ For example:
+			§ LC_driver_WinServer2016\WinServer2016\Signed_1152921504627955523.zip needs to be unzipped and the drivers\269d7f2d-bda1-4ce8-8f0b-7d65f8a0170e\sgx_base_dev.inf it contains installed to root\SgxLCDevice.
+			§ DCAP_INF\WinServer2016\Signed_1152921504627956768.zip also needs to be unzipped and the drivers\27e5c4c3-2f1b-45b0-abb7-234a0df70a5b\sgx_dcap_dev.inf it contains installed to root\SgxLCDevice_DCAP.
+			§ Both the nuget packages nupkg\DCAP_Components\DCAP_Components.1.1.100.1.nupkg and nupkg\EnclaveCommonAPI\EnclaveCommonAPI.1.0.100.0.nupkg need to be installed as well.
+		○ To enable the LC driver, the system regkey must also be set: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters\"SGX_Launch_Config_Optin"=dword:00000001
+			§ This requires a reboot to take effect once the driver is running.
+	• Install the Azure DCAP client nupkg
+		○ Install Windows10 SDK.17134 which is a pre-requisite to build.
+		○ There's no pre-built drop of this from https://github.com/Microsoft/Azure-DCAP-Client/tree/master/src/Windows currently, so manually build it using the instructions from the README.md file. There are 2 ways to build this using Visual Studio 2017 or using the provided scripts.
+
+	• To install driver .inf from command line:
+	   ○ devcon.exe from the Windows Driver Kit for Windows 10 can be used.
+      For example:
+         devcon.exe install sgx_base_dev.inf root\SgxLCDevice
+         devcon.exe install sgx_dcap_dev.inf root\SgxLCDevice_DCAP
+	• It is usually installed to C:\Program Files (x86)\Windows Kits\10\tools\x64 and is not pathed by default.
+
+	• To install .nupkg from command line
+	   ○ nuget.exe (https://dist.nuget.org/win-x86-commandline/latest/nuget.exe) for CLI can be used.
+      For example: (Add nuget.exe to your PATH or provide full path)
+	      nuget.exe install EnclaveCommonAPI -Source "C:\Packages\Intel SGX DCAP for Windows v1.1.100.49925\nupkg" -OutputDirectory C:\openenclave\prereqs\nuget
+	      nuget.exe install DCAP_Components -Source "C:\Packages\Intel SGX DCAP for Windows v1.1.100.49925\nupkg" -OutputDirectory C\openenclave\prereqs\nuget
+         nuget.exe install Microsoft.Azure.DCAP.Client -Source "C:\Packages\Azure-DCAP-Client\src\Windows;nuget.org" -OutputDirectory C:\dcap_latest\openenclave\prereqs\nuget
+
+	• Finally, strip the version numbers from the DCAP libraries as follows: (assuming that your openenclave directory is at c:\openenclave)
+      Rename C:\openenclave\prereqs\nuget\Intel SGX DCAP for Windows v1.1.100.49925 to C:\openenclave\prereqs\nuget\Intel SGX DCAP for Windows
+      Rename C:\openenclave\prereqs\nuget\EnclaveCommonAPI.1.0.100.0 to C:\openenclave\prereqs\nuget\EnclaveCommonAPI
+      Rename C:\openenclave\prereqs\nuget\Microsoft.Azure.DCAP.Client.1.0.0 to C:\openenclave\prereqs\nuget\Microsoft.Azure.DCAP.Client
+
```

Comment:
Are we sure we need all these manual steps? It sounds like way more complicated than the Linux experience.

Reply:
Ain't that always the truth on Windows 😆

Reply:
Yes, these are the manual steps and will hopefully, get better soon.
Semi-automation to deploy an ACC Win VM for Windows with DCAP libraries using oe-engine/ansible scripts is documented on our wiki page - will send out the link to the team separately.
We also have a new script to automate the Azure DCAP client on Windows build. Will update this document with that script. Hoping that this build will be readily available in the future.